application/testing/CMakeLists.txt

```diff
@@ -607,6 +607,7 @@ f3d_test(NAME TestNoFileConfigFile CONFIG ${CMAKE_SOURCE_DIR}/testing/configs/ve
 
 # Test help display
 f3d_test(NAME TestHelp ARGS --help REGEXP "Usage:")
+f3d_test(NAME TestHelpPositional ARGS --help REGEXP "<input>")
```

Comment:
should test for `--input` instead, positional arguments were already in the help (`file1 file2 ...`), what this PR is adding is positional arguments *as named arguments*